### PR TITLE
Yaml query util

### PR DESCRIPTION
**Pull request by ahmed-mez**
### What does this PR do?

Add yaml query util

### Motivation

Compliance ™️

### CODE REVIEW

**Review by xornivore — APPROVED** [review]
👍

File: pkg/compliance/checks/file_check.go [hunk 1]
```diff
@@ -112,28 +116,37 @@ func (c *fileCheck) getAttribute(filePath string, fi os.FileInfo, property strin
 	return "", ErrPropertyNotSupported
 }
 
-func (c *fileCheck) getJSONPathValue(filePath string, jsonquery string) (string, error) {
-	f, err := os.Open(filePath)
-	if err != nil {
+type getter func([]byte, string) (string, error)
```
Comment (xornivore) [inline pkg/compliance/checks/file_check.go:119]: Nit - might make sense to add a comment here even though it isn't exported.

File: pkg/util/json/jsonquery.go [hunk 2]
```diff
@@ -15,8 +15,8 @@ import (
 )
 
 const (
-	cacheTTL      time.Duration = 6 * time.Hour
-	jqCachePrefix string        = "jq-"
+	cacheTTL      = 6 * time.Hour
```
Comment (xornivore) [inline pkg/util/json/jsonquery.go:18]: 👌

**Review by xornivore — APPROVED** [review]
🚀 , had a small note on adding comment, but looks great otherwise.

File: pkg/compliance/checks/file_check.go (hunk 1 above)
Comment (ahmed-mez) [reply pkg/compliance/checks/file_check.go:120]: done 💯

**Pull request merged**